[2024-09-15T20:33:29.221+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-15T20:27:42.693756+00:00 [queued]>
[2024-09-15T20:33:29.230+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-15T20:27:42.693756+00:00 [queued]>
[2024-09-15T20:33:29.230+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 12
[2024-09-15T20:33:29.242+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): raw_ingestion> on 2024-09-15 20:27:42.693756+00:00
[2024-09-15T20:33:29.246+0000] {standard_task_runner.py:57} INFO - Started process 184 to run task
[2024-09-15T20:33:29.248+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'raw_ingestion', 'manual__2024-09-15T20:27:42.693756+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmphjluxf8r']
[2024-09-15T20:33:29.249+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask raw_ingestion
[2024-09-15T20:33:29.340+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-15T20:27:42.693756+00:00 [running]> on host airflow-worker
[2024-09-15T20:33:29.488+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='raw_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2024-09-15T20:27:42.693756+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-15T20:27:42.693756+00:00'
[2024-09-15T20:33:29.489+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-15T20:33:29.489+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-15T20:33:29.514+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-15T20:33:29.515+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-15T20:33:29.523+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-15T20:33:29.605+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-15T20:33:29.605+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py
[2024-09-15T20:33:31.226+0000] {ssh.py:524} INFO - Traceback (most recent call last):
[2024-09-15T20:33:31.226+0000] {ssh.py:524} INFO -   File "/opt/shared/scripts/raw_ingestion.py", line 1, in <module>
[2024-09-15T20:33:31.228+0000] {ssh.py:524} INFO -     import requests
[2024-09-15T20:33:31.228+0000] {ssh.py:524} INFO - ModuleNotFoundError: No module named 'requests'
[2024-09-15T20:33:31.236+0000] {ssh.py:524} INFO - 24/09/15 20:33:31 INFO ShutdownHookManager: Shutdown hook called
[2024-09-15T20:33:31.238+0000] {ssh.py:524} INFO - 24/09/15 20:33:31 INFO ShutdownHookManager: Deleting directory /tmp/spark-d33853d7-23c2-4d03-bd9e-556b37dc1b76
[2024-09-15T20:33:31.321+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 166, in run_ssh_client_command
    self.raise_for_status(exit_status, agg_stderr, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 160, in raise_for_status
    raise AirflowException(f"SSH operator error: exit status = {exit_status}")
airflow.exceptions.AirflowException: SSH operator error: exit status = 1
[2024-09-15T20:33:31.323+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=raw_ingestion, task_id=raw_ingestion, execution_date=20240915T202742, start_date=20240915T203329, end_date=20240915T203331
[2024-09-15T20:33:31.338+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 47 for task raw_ingestion (SSH operator error: exit status = 1; 184)
[2024-09-15T20:33:31.386+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-15T20:33:31.422+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
